server/util/current_request/current_request.go

```diff
@@ -0,0 +1,12 @@
+package current_request
```

Comment:
nit: maybe call this client_ip and change `func ClientIP` to `func Get`

Reply:
(or move other current_request stuff here if you think that's a better fit)

Reply:
renamed to clientip.